### ISSUE TITLE
a11y - autocomplete attributes in text fields

### ISSUE THREAD

**Issue by sabrina-bongiovanni** [collective/volto-form-block#117]
The Form block does not have any "autocomplete" attribute which goes against  [[WCAG 2.1 (AA)]Identify the purpose of inputs using the autocomplete value](https://www.w3.org/WAI/WCAG21/Techniques/html/H98)

It should be possible to add the autocomplete attribute in edit mode when setting up the block so that whoever adds it to a page can use the correct one.
